DTS-STN/Service-Canada-Design-System
# Fix footer background overflow

# DS-##Footer.js

Main Reviewer: @zeddotes 

#### Description & Background Context:

Fixed issue where the grey background was still being used in the top <section> of the auth footer. 

#### Where should the reviewer start?

File name: Footer.js

#### How should this be tested?

View component in storybook, see that there is no grey background colour overflowing from the first footer <section> (the blue section.)

#### Are there any breaking changes?
